lib/yard/code_objects/constant_object.rb

```diff
@@ -12,5 +12,12 @@ class ConstantObject < Base
     def value=(value)
       @value = format_source(value)
     end
+
+    # The scope of the constants is always +:class+
+    #
+    # @return [Symbol] +:class+
+    def scope
```

Comment:
If there are no groups, but you have constants it raises an exception `NoMethodError: undefined method `scope'` like it did with the classvariable in the failing test altough I'm not sure why this is triggered but I'll definitely look into it.